[quantization] fix conv2d

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #23702 [probably wip] Quantization tests in OSS
* #23712 Maybe fix qat import
* **#23690 [quantization] fix conv2d**

Differential Revision: [D16610734](https://our.internmc.facebook.com/intern/diff/D16610734)

Commit:
[quantization] fix conv2d

jerryzh168 commented on torch/nn/quantized/functional.py:
```diff
@@ -115,12 +115,12 @@ def conv2d(input, weight, bias,
-                                       prepacked_weight, bias,
-                                       stride, padding, dilation,
-                                       groups, scale, zero_point).permute([0, 3, 1, 2])
+    return _ops.quantized.fbgemm_conv2d(input.permute([0, 2, 3, 1]),
```
maybe remove `as _ops` in the import?

zafartahirov replied:
The reason it was there was so that the user wouldn't have `ops` in their top level after `from torch.quantized.functional import *`

jerryzh168 replied:
maybe use torch.ops directly then?

jamesr66a replied:
Does it make a difference? @zafartahirov  do you know?

Commit:
Update on "[quantization] fix conv2d"

[quantization] fix conv2d

gh-metadata: pytorch pytorch 23690 gh/jamesr66a/28/head
